server/package.json

```diff
@@ -4,6 +4,8 @@
   "description": "",
   "main": "index.ts",
   "scripts": {
+    "build": "tsc -p .",
+    "serve:local": "cross-env NODE_ENV=development nodemon index.ts",
     "serve": "nodemon index.ts",
     "migrate": "node-pg-migrate",
     "try": "echo %EnvVar%"
```

Comment:
Btw, what is this script for?

Reply:
It was a try to use a different .env file depeding on development but finally I just decided to add the local .env file to the dockerignore in order to not override the .env specified file in the .yml. I'm goint to delete it

Reply:
Could you remove this script, please :)

Reply:
already done, forgot to write it down :)

Reply:
I'm still able to see it :thinking:

Reply:
I can't see it anymore, maybe that piece of code is an old version???

Reply:
It's still here https://github.com/jblanava/Pern-test/blob/0a48ef4ff171b8eb89142e6064eb4e5bf8ecb8b6/server/package.json#L9